controllers/tracks.ts

```diff
@@ -10,12 +11,46 @@ import asyncHandler from "express-async-handler";
 import { parseLimit } from "../library/helpers";
 import { AWS_S3_RAW_PREFIX, AWS_S3_TRACK_PREFIX } from "../library/constants";
 import { addOP3URLPrefix } from "../library/op3";
+import {
+  FEATURED_PLAYLIST_ID,
+  DEFAULT_FOR_YOU_PLAYLIST_ID,
+} from "../library/constants";
+import { getPlaylistTracks } from "../library/playlist";
+import { getWeeklyTop40 } from "../library/chart";
+import { getUserRecentTracks, getNewTracks } from "../library/track";
 
 const randomSampleSize = process.env.RANDOM_SAMPLE_SIZE;
-
 const s3BucketName = `${process.env.AWS_S3_BUCKET_NAME}`;
 const cdnDomain = `${process.env.AWS_CDN_DOMAIN}`;
 
+const get_featured_tracks = asyncHandler(async (req, res, next) => {
+  let pubkey;
+  pubkey = res?.locals?.authEvent as Event;
+
+  const featuredTracks = await getPlaylistTracks(FEATURED_PLAYLIST_ID);
+  const trendingTracks = await getWeeklyTop40();
+  const newTracks = await getNewTracks();
+  let forYouTracks;
+  if (pubkey) {
+    log.debug(`Nostr pubkey: ${pubkey}`);
+    forYouTracks = await getUserRecentTracks(pubkey);
+    if (forYouTracks.length < 3) {
+      forYouTracks = await getPlaylistTracks(DEFAULT_FOR_YOU_PLAYLIST_ID);
+    }
+  } else {
+    forYouTracks = await getPlaylistTracks(DEFAULT_FOR_YOU_PLAYLIST_ID);
+  }
+  res.send({
+    success: true,
+    data: {
+      featured: featuredTracks,
+      forYou: forYouTracks,
+      trending: trendingTracks,
+      new: newTracks,
```

Comment:
VSC is reminding me `new` is a reserved keyword in javascript
```suggestion
      newTracks: newTracks,
```